extensions/amp-auto-ads/0.1/test/test-placement.js

```diff
@@ -42,6 +42,10 @@ describes.realWin('placement', {
     env.win.document.body.appendChild(container);
   });
 
+  afterEach(() => {
+    sandbox.restore();
```

Comment:
`env.sandbox` should already be a sandbox, no need to create a new or restore it.